docs/GroupDefs.hpp

```diff
@@ -442,6 +442,11 @@
  * <td> \ref ParallelInfoExecutablePage "ParallelInfo" </td>
  * <td> Executable for checking number of nodes, cores, etc.</td>
  * </tr>
+ * <tr>
+ * <td> DumpEquationOfStateForRotNS </td>
```

Comment:
Use "export" as a nicer word for "dump", and consistent with ExportCoordinates?

Reply:
Sure!